systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java

```diff
@@ -2128,6 +2137,344 @@ void testKafkaOffsetsReplicationFactorHigherThanReplicas() {
+
+        BasicExternalKafkaClient basicExternalKafkaClientTls = new BasicExternalKafkaClient.Builder()
+                .withTopicName(TOPIC_NAME)
+                .withNamespaceName(NAMESPACE)
+                .withClusterName(CLUSTER_NAME)
+                .withMessageCount(MESSAGE_COUNT)
+                .withConsumerGroupName(CONSUMER_GROUP_NAME + "-" + rng.nextInt(Integer.MAX_VALUE))
+                .withSecurityProtocol(SecurityProtocol.SSL)
+                .build();
+
+        BasicExternalKafkaClient basicExternalKafkaClient = new BasicExternalKafkaClient.Builder()
+                .withTopicName(TOPIC_NAME)
+                .withNamespaceName(NAMESPACE)
+                .withClusterName(CLUSTER_NAME)
+                .withMessageCount(MESSAGE_COUNT)
+                .withConsumerGroupName(CONSUMER_GROUP_NAME + "-" + rng.nextInt(Integer.MAX_VALUE))
+                .withSecurityProtocol(SecurityProtocol.PLAINTEXT)
+                .build();
+
+        KafkaResource.kafkaEphemeral(CLUSTER_NAME, kafkaReplicas, 1)
+                .editSpec()
+                    .editKafka()
+                        .withNewListeners()
+                            .withNewKafkaListenerExternalNodePort()
+                                //.withTls(false)
+                            .endKafkaListenerExternalNodePort()
+                            .withNewPlain()
+                            .endPlain()
+                        .endListeners()
+                        .withConfig(kafkaConfig)
+                    .endKafka()
+                .endSpec()
+                .done();
+
+        KafkaTopicResource.topic(CLUSTER_NAME, TOPIC_NAME).done();
+
+        String userName = "alice";
+        KafkaUserResource.tlsUser(CLUSTER_NAME, userName).done();
+        waitFor("Wait for secrets became available", Constants.GLOBAL_POLL_INTERVAL, Constants.TIMEOUT_FOR_GET_SECRETS,
+            () -> kubeClient().getSecret(userName) != null,
+            () -> LOGGER.error("Couldn't find user secret {}", kubeClient().listSecrets()));
+
+        String userName2 = "john";
+        KafkaUserResource.tlsUser(CLUSTER_NAME, userName2).done();
+        waitFor("Wait for secrets became available", Constants.GLOBAL_POLL_INTERVAL, Constants.TIMEOUT_FOR_GET_SECRETS,
+                () -> kubeClient().getSecret(userName2) != null,
+                () -> LOGGER.error("Couldn't find user secret {}", kubeClient().listSecrets()));
+
+        basicExternalKafkaClientTls.setKafkaUsername(userName);
+        basicExternalKafkaClient.setKafkaUsername(userName2);
+
+        assertThrows(ExecutionException.class, () -> {
+            Future<Integer> failproducer = basicExternalKafkaClient.sendMessagesPlain(5000);
+            Future<Integer> failconsumer = basicExternalKafkaClient.receiveMessagesPlain(5000);
+
+            assertThat(failproducer.get(2, TimeUnit.MINUTES), is(MESSAGE_COUNT));
+            assertThat(failconsumer.get(2, TimeUnit.MINUTES), is(MESSAGE_COUNT));
+        });
```

Comment:
Use our constants from the `Constants class` where you will find GLOBAL_CLIENTS_TIMEOUT.... and please avoid using this hard-code values.